example/lib/page/save_image_example.dart

```diff
@@ -38,11 +38,12 @@ class _SaveMediaExampleState extends State<SaveMediaExample> {
     if (Platform.isIOS || Platform.isMacOS) {
       dir = (await getApplicationSupportDirectory()).absolute.path;
     } else if (Platform.isAndroid) {
-      dir = (await getExternalStorageDirectories(
-        type: StorageDirectory.downloads,
-      ))![0]
-          .absolute
-          .path;
+      // dir = (await getExternalStorageDirectories(
```

Comment:
Remove this?

Reply:
these two `dir` path represent two different way in `DBUtils`->`saveImage` 
```
if (savePath) { //true or false
    inputStream.close()
} else {
 ...
```
so, I suggest keep this

Reply:
No, I mean the commented out part.

Reply:
I changed it back, since the `saveVideo` method has acquired `getTemporaryDirectory` path, so it can cover this two cases